packages/preview/vienna-tech/1.1.0/src/constructors.typ

```diff
@@ -0,0 +1,182 @@
+    (1fr,) * 3
+  }
+
+  align(center, {
+    set text(font: title-font, size: large-size)
+    v(50pt, weak: false)
+    repr-thesis-type
+    v(25pt, weak: true)
+    repr-thesis-title
+    v(25pt, weak: true)
+
+    set text(size: font-size)
+    grid(
+      columns: cols, align: center,
+      ..authors,
+    )
+    if date != none {
+      align(center)[#date-prefix #date]
+    }
+  })
+}
+
+/// This function constructs the abstract, which is supposed to come directly after the frontmatter.
+/// -> content
+#let abstract(
+  content,
+  /// The font, which is only used for the heading of the abstract paragraph.
+  /// -> string
+  font: title-font,
+  /// The title which the abstract paragraph should have.
+  /// -> content | string
+  title: [Kurzfassung],
+) = {
+  // English abstract
+  v(50pt, weak: true)
+  set text(small-size)
+  show: pad.with(x: 1cm)
+  align(center, text(font: font, strong(title)))
+  v(6pt, weak: true)
+  content
+}
+
+/// This function will help you to provide a long caption to a figure,
+/// but a short caption to the outline.
+///
+/// Example usage:
+/// ```typ
+/// #figure(
+///   rect(),
+///   caption: shortcap("Short caption", "Long caption"),
+/// )
+/// ```
+///
+/// -> content
+#let shortcap(
+  /// The short caption of the figure.
+  /// -> string | content
+  short,
+  /// The long caption of the figure.
+  /// -> string | content
+  long,
+) = context if outlined.get() { short } else { long }
+
+
+/// This function is a wrapper of the `outline` function and allows for an easy
+/// way to create the table of figures.
+///
+/// Example usage:
+/// ```typ
+/// #fig-outline()
+/// ```
+/// -> content
+#let fig-outline(
+  /// The title of the table of figures.
+  /// -> string | content
+  title: "List of Figures",
+  /// The Target of the table of figures.
+  /// -> label | selector | location | function
+  target: figure.where(kind: image),
+  /// Additional optional arguments to the outline function.
+  ..args,
+) = outline(target: target, title: title, ..args)
+
+/// This function is a wrapper of the `outline` function and allows for an easy
+/// way to create the list of tables.
+///
+/// Example usage:
+/// ```typ
+/// #tab-outline()
+/// ```
+/// -> content
+#let tab-outline(
+  /// The title of the table of figures.
+  /// -> string | content
+  title: "List of Tables",
+  /// The Target of the table of figures.
+  /// -> label | selector | location | function
+  target: figure.where(kind: table),
+  /// Additional optional arguments to the outline function.
+  ..args,
+) = outline(target: target, title: title, ..args)
+
+/// This function is a dictionary that contains the configuration
+/// for an Author. Any attributes can be added to the dictionary.
+/// The default attributes are `name`, `email`, and `matrnr` and are needed for the default `maketitle` function. But these can be modified or extended, via the fields and field-prefixes arguments.
+/// -> dict
+#let Author(
```

Comment:
Unless there is a good reason for preferring `Author`, I think this function should be called `author` without a capital letter, for more consistency with the other ones.